frontend/src/views/intro-view.js

```diff
@@ -12,14 +14,95 @@ class IntroView extends PolymerElement {
 
     static get template() {
         return html`
-            <h1>Layout Examples for Vaadin 14</h1>
-            <p>Pick a layout from the menu.</p>
+            <style include="common-styles">
+                :host {
+                    display: block;
+                }
+                
+                .wrapper {
```

Comment:
There should be only one `main` tag on the page, which is the case for this view as well.
So I think it is safe to remove `.wrapper` class from it and style using tag name.

Reply:
Done.